automated-comments/javascript/two-fer/redirect_incorrect_string_template.md

```diff
@@ -0,0 +1,2 @@
+<!-- only for refer_to_mentor -->
+The string template looks incorrect. Expected a template with 3 components.
```

Comment:
```diff
- Expected a template with 3 components.
+ A template with 3 components was expected.
```

Reply:
To me these are equal 😅 . Making the change ⚡️ 

```suggestion
The string template looks incorrect. A template with 3 components was expected.
```

Reply:
Yes, the same, but the missing subject before "expected" is taken care of with the suggested change.  I will often leave off the "we" or "I" in my writing, it is not correct, but it is understood (with ambiguity) and is a bad habit.